packages/broker/src/ConfigWizard.ts

```diff
@@ -0,0 +1,195 @@
+                message: "'Please provide the private key to import'",
+            })
+
+            try {
+                const wallet = new Wallet(privateKey)
+                this.config.ethereumPrivateKey = wallet.privateKey
+            } catch (privateKeyError){
+                throw new Error(`Invalid privateKey provided for import: ${privateKey}`)
+            }
+
+        } else {
+            throw new Error(`Invalid option ${generateOrImport} provided`)
+        }
+
+        return this.config.ethereumPrivateKey
+    }
+
+    async promptNumberWithDefault(prompt: inquirer.InputQuestion, defaultValue: number){
+        const valueString = await this.inquirerSinglePrompt(prompt)
+        let value = parseInt(valueString)
+        if (valueString === ''){
+            value = defaultValue
+        }
+
+        if (!Number.isNaN(value) && Number.isInteger(value) && value > 1023 && value < 49151){
+            return value
+        } else {
+            return defaultValue
+        }
+    }
+
+    async selectPlugins(): Promise<Array<string>>{
+        const plugins: inquirer.Answers = await this.inquirerSinglePrompt( {
+            type: 'checkbox',
+            name:'selectedItems',
+            message: 'Select the plugins to enable',
+            choices: [
+                {name: 'Websocket'},
+                {name:'MQTT'},
+                {name:'HttpPublish'}
+            ]
+        })
+
+        const selectedPlugins = []
+        for (let i = 0; i < plugins.length; i++){
+            selectedPlugins.push(plugins[i])
+            if (plugins[i] === 'Websocket'){
+                const wsPort = await this.promptNumberWithDefault({
+                    type: 'input',
+                    name: 'wsPort',
+                    message: `Select a port for the Websocket Plugin [Enter for default: ${this.defaultWebsocketPort}]`,
+                }, this.defaultWebsocketPort)
+                this.config.plugins['websocket'] = { port: wsPort }
+            }
+
+            if (plugins[i] === 'MQTT'){
+                const mqttPort = await this.promptNumberWithDefault({
+                    type: 'input',
+                    name: 'mqttPort',
+                    message: `Select a port for the MQTT Plugin [Enter for default: ${this.defaultMqttPort}]`,
+                }, this.defaultMqttPort)
+                this.config.plugins['mqtt'] = { port: mqttPort }
+            }
+
+            if (plugins[i] === 'HttpPublish'){
+                const httpPort = await this.promptNumberWithDefault({
+                    type: 'input',
+                    name: 'httpPort',
+                    message: `Select a port for the HttpPublish Plugin [Enter for default: ${this.defaultHttpPort}]`,
+                }, this.defaultHttpPort)
+                this.config.plugins['httpPublish'] = { port: httpPort }
+            }
+        }
+
+        return selectedPlugins
+
+    }
+
+    async storeConfig(destinationFolder = '../configs/'){
```

Comment:
Let's move the default value up to the exported function for clarity.
```suggestion
    async storeConfig(destinationFolder: string){
```